kits/apptentive-kit/src/main/java/com/mparticle/kits/ApptentiveKit.java

```diff
@@ -0,0 +1,243 @@
+package com.mparticle.kits;
+
+import android.app.Activity;
+import android.app.Application;
+import android.location.Location;
+import android.text.TextUtils;
+
+import com.apptentive.android.sdk.Apptentive;
+import com.apptentive.android.sdk.ApptentiveInternal;
+import com.apptentive.android.sdk.lifecycle.ApptentiveActivityLifecycleCallbacks;
+import com.apptentive.android.sdk.model.CommerceExtendedData;
+import com.mparticle.MPEvent;
+import com.mparticle.MParticle;
+import com.mparticle.commerce.CommerceEvent;
+import com.mparticle.commerce.Product;
+import com.mparticle.commerce.TransactionAttributes;
+import com.mparticle.internal.CommerceEventUtil;
+import com.mparticle.internal.ConfigManager;
+
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+public class ApptentiveKit extends AbstractKit implements ClientSideForwarder, ECommerceForwarder, ActivityLifecycleForwarder {
+	private static final String API_KEY = "appKey";
+	private ApptentiveActivityLifecycleCallbacks callbacks;
+
+	@Override
+	protected AbstractKit update() {
+		if (callbacks == null) {
+			callbacks = new ApptentiveActivityLifecycleCallbacks(context.getApplicationContext());
```

Comment:
Will the `ApptentiveActivityLifecycleCallbacks` object use the static methods in Apptentive?